ckanext/canada/controller.py

```diff
@@ -458,11 +458,18 @@ def output_feed(self, results, feed_title, feed_description,
             last_page=navigation_urls['last'],
         )
 
+        chromos = h.recombinant_get_types()
+
         for pkg in results:
+            title, notes = None, None
+            if pkg['type'] in chromos:
+                chromo = h.recombinant_get_chromo(pkg['type'])
+                title= _(chromo['title'])
+                notes = _(pkg['notes'])
```

Comment:
It is not there. See http://registry.open.canada.ca/en/api/2/rest/package/c3d6f6a9-9558-4376-9237-4573aee5f835